### DISCUSSION TITLE
Failing Builds iOS React Native

### DISCUSSION THREAD

**Discussion by saintmalik** [codemagic-ci-cd/codemagic-docs#2135] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

We didn't make any changes but the build started failing, it builds locally but not in the CI, here is the logs link: https://codemagic.io/app/637bd153143bfdd9801b2dd7/build/6532ce30c5de6f65048b8031

### Expected results

Successful build

### Actual results

Errors

**Comment by himesh-cm** [comment]
Hi @saintmalik, the last successful build was on Xcode 14.3.1, while the failed one was on Xcode 15.0. You can switch to 14.3.1 for a temporary workaround. There are compilation errors. Have you tried building locally with Xcode 15.0?

**Comment by himesh-cm** [comment]
Hi @saintmalik,
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
